### PR TITLE
major cleanup of the gulpfile

### PR DESCRIPTION
**Pull request by jdungan**
Testing widget on FF nightly and had to:
- clean up creation of xpi file for dev and prod
- consolidate redundant ff and chrome code (should make opera easier to add)
- eliminate some leftovers from the foundation change
- update readme

### CODE REVIEW

File: src/home.coffee [hunk 1]
```diff
@@ -21,18 +21,15 @@ if isChrome
...
   codesy.save = (home)->
     console.log "save domain: " + home.domain
     chrome.runtime.sendMessage home
 
-$token = $("#api_token_pass")
-
 codesy_home =
   'task' : "setHome"
   'domain': window.location.origin
-  'token': $token.val() or ""
```
Comment (jdungan) [inline src/home.coffee]: We no longer need to pass an auth token for ajax calls

File: src/home.coffee [hunk 2]
```diff
@@ -21,18 +21,15 @@ if isChrome
...
       data.domains.unshift(home)
       chrome.storage.local.set(data);
-
+      
 else # firefox
   codesy.save = (home)->
     console.log "save domain: " + home.domain
     chrome.runtime.sendMessage home
 
-$token = $("#api_token_pass")
```
Comment (jdungan) [inline src/home.coffee]: This files now that we no longer need to pass an auth token for ajax calls

**Comment by groovecoder** [comment]
Looks good and works well. Except why is there `chrome-coffee` and `chrome-static` - none of those are chrome-specific now, and I'm not even sure how it's working; trying to source the `.coffee` files from `src/chrome` when they just live in `src/` now?

**Comment by groovecoder** [comment]
Commit should say "fix #82: "
codesy-ff-nightly
+:100:

**Pull request merged**